packages/fc-versions/firecracker_versions.txt

```diff
@@ -0,0 +1,2 @@
+v1.5.0
+8bb883111ff608dc2ca1a08d5e4eb2ddbd5c1c8b
```

Comment:
it would be nice to have some comments to it, otherwise it's hard to read

Reply:
I added a readme where I explained the whole package together with the versioning system.